Formula/openblas.rb

```diff
@@ -22,6 +22,8 @@ def install
     ENV["DYNAMIC_ARCH"] = "1"
     ENV["USE_OPENMP"] = "1"
     ENV["NO_AVX512"] = "1"
+    # Force a large NUM_THREADS to support larger Macs than the VMs that build the bottles
+    ENV["NUM_THREADS"] = "56"
```

Comment:
Is this the highest available on macs?

Reply:
I believe so. The highest-core-count Mac Pro is currently a 28-core Xeon (for 56 threads; 2 per hardware core); I don't know of any Mac with a higher core count.

Or do you mean how high could we possibly set it, regardless of what Mac hardware is actually available? We could probably go a lot higher, at the cost of additional memory usage for the per-thread buffer structures in OpenBLAS. I don't know how much memory those actually take.

Reply:
I did a build with `NUM_THREADS` = 256, and Octave is still taking 26 MB when running against it.

Reply:
Let's get ready for the future 😄

Reply:
I've bumped it to 256. Can't wait to see a Mac that needs more than that...

Reply:
For the internal structures, they're not big in the sense of "occupying a lot of RAM" but having them too large means you have poor cache locality in performance-crucial code (tight loops, indirections, etc). I'd suggest using what we need (56), but not actually more, because the performance hit can actually be important.

Reply:
Oh, that makes a lot of sense.

I have dialed it back down to 56 threads, and will defer to y'all's judgment on further adjustments needed here.

I also bumped the revision, which I neglected to do originally.